flexmeasures/conftest.py

```diff
@@ -1038,3 +1067,67 @@ def error_generator():
     @roles_accepted(ADMIN_ROLE)
     def vips_only():
         return jsonify({"message": "Nothing bad happened."}), 200
+
+
+@pytest.fixture(scope="module")
+def capacity_sensors(db, add_battery_assets, setup_sources):
+    battery = add_battery_assets["Test battery with dynamic power capacity"]
+    production_capacity_sensor = Sensor(
+        name="production capacity",
+        generic_asset=battery,
+        unit="MW",
+        event_resolution="PT15M",
+        attributes={"consumption_is_positive": True},
+    )
+    consumption_capacity_sensor = Sensor(
+        name="consumption capacity",
+        generic_asset=battery,
+        unit="MW",
```

Comment:
For testing, it could be interesting to set up one of the capacity sensors with a different unit, like kW.